docs/cody/capabilities/supported-models.mdx

```diff
@@ -12,6 +12,7 @@ Cody supports a variety of cutting edge large language models for use in Chat an
 | OpenAI       | [gpt-4o](https://platform.openai.com/docs/models/gpt-4o) | -              | ✅              | ✅              |
 | Anthropic    | [claude-3 Haiku](https://docs.anthropic.com/claude/docs/models-overview#model-comparison)                                                     | -              | ✅              | ✅              |
 | Anthropic    | [claude-3 Sonnet](https://docs.anthropic.com/claude/docs/models-overview#model-comparison)                                                    | ✅              | ✅              | ✅              |
+| Anthropic    | [claude-3.5 Sonnet](https://docs.anthropic.com/claude/docs/models-overview#model-comparison)                                                    | ✅              | ✅              | -              |
```

Comment:
Looks good! FYI Enterprise support should be ready in the next Sourcegraph release too 😄